packages/login/package.json

```diff
@@ -88,6 +89,7 @@
     "eslint-plugin-flowtype": "^5.2.0",
     "eslint-plugin-jsx-a11y": "^6.3.1",
     "eslint-plugin-react-hooks": "^4.0.8",
+    "google-libphonenumber": "^3.2.32",
```

Comment:
@rikukissa should this not be in dependencies as opposed to devDependencies if it is to build properly in Docker?